docs/docs-content/clusters/edge/upgrade-behavior.md

```diff
@@ -12,6 +12,20 @@ profile. Depending on the nature of the change, implementing an upgrade might in
 cluster, restarting services, or doing nothing. For more information about cluster repaves, refer to
 [Repave Behavior and Configurations](../cluster-management/node-pool.md#repave-behavior-and-configuration).
 
+:::warning
+
+- If a cluster's Kubernetes service is down, updates to the cluster's profile will not get applied. You must fix the
+  issue impacting the Kubernetes service first and ensure that the Kubernetes service of the cluster is operational
+  before applying updates to the cluster.
```

Comment:
Should this be a list item? 👀

Reply:
I'm a bit confused with the note: If the k8s service is down, Palette will not apply updates from the cluster profile to the cluster? Or updates to the cluster profile itself? Then, once you fix the k8s service, do you have to manually apply the cluster profile updates? I think this is where I'm lost:

- Who/what doesn't apply the updates if the k8s service is down?
- What is being updated, the deployed cluster or the cluster profile?
- What updates should the used apply once the k8s service is up again?

Reply:
It applies to updates to the cluster. When you update a profile, the updates in the profile get applied to the cluster. If K8s is down, the updates won't be applied to the cluster.